tests/e2e/gc_block_volume_expansion.go

```diff
@@ -122,6 +125,17 @@ var _ = ginkgo.Describe("[csi-guest] Volume Expansion Test", func() {
 		restConfig = getRestConfigClient()
 		isGCCSIDeploymentPODdown = false
 
+		// Get GC CSI Replica Count
+		gcDeployment, err := client.AppsV1().Deployments(csiSystemNamespace).Get(ctx,
+			vSphereCSIControllerPodNamePrefix, metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gcCsiReplicaCountBeforeScaleDown = *gcDeployment.Spec.Replicas
+
+		// Get SVC CSI Replica Count
+		svcDeployment, err := svcClient.AppsV1().Deployments(csiSystemNamespace).Get(ctx,
+			vSphereCSIControllerPodNamePrefix, metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		svcCsiReplicaCountBeforeScaleDown = *svcDeployment.Spec.Replicas
```

Comment:
This is not correct IMO, if a previous test case had crashed in the middle and replicas were changed in it, we will not be using correct values. Let's use a constant set to 3.